tox.ini

```diff
@@ -25,7 +25,7 @@ skip_install = true
 usedevelop = true
 envdir = {toxworkdir}/py37
 commands =
-    poetry install -v
+    poetry install -v -E mlflow
```

Comment:
Why do you mlflow here?

Reply:
I added a test in which train uses mlflow. Hits all the `MLflowLogger` code :)